tests/unit/conftest.py

```diff
@@ -232,7 +234,54 @@ def units():
 
 
 @pytest.fixture
-def apps(status, config):
+def config():
+    return {
+        "openstack_ussuri": {
+            "openstack-origin": {"value": "distro"},
+            "action-managed-upgrade": {"value": True},
+        },
+        "openstack_wallaby": {"openstack-origin": {"value": "cloud:focal-wallaby"}},
+        "auxiliary_ussuri": {"source": {"value": "distro"}},
+        "auxiliary_wallaby": {"source": {"value": "cloud:focal-wallaby"}},
+    }
+
+
+async def get_status():
```

Comment:
We probably will need to be able to get status from more than one json file. I think this function should be ready to receive the file name as parameter

Reply:
I agree, but this could be done in next PR when we actually use it.